mitodl/ocw-studio
# add migration to move filetype to resourcetype

#### Pre-Flight checklist


#### What are the relevant tickets?
Prerequisite for https://github.com/mitodl/ocw-hugo-projects/issues/160

#### What's this PR do?
While https://github.com/mitodl/ocw-hugo-projects/pull/161 has been in review, it came to light that the `ocw-www` starter configuration has become a bit out of sync with some changes we've been making in `ocw-course`.  We were setting up linking / embedding of resources and realized that the filtering on resource type wasn't working properly.  This is because this field is currently called `filetype` in `ocw-www` when it should be called `resourcetype` because that's the field that `ocw-studio` looks at when using its filter logic.  This PR introduces a migration that moves `filetype` to `resourcetype` in resources that are part of `ocw-www` specifically.

#### How should this be manually tested?
 - Spin up `ocw-studio` locally and make sure you have an `ocw-www` site populated with some content, including resources
 - Run `docker-compose run --rm web ./manage.py migrate`
 - Go to Django admin and verify on a resource that's part of `ocw-www` that `filetype` has been migrated to `resourcetype`
